source/extensions/common/wasm/plugin.cc

```diff
@@ -52,6 +52,16 @@ WasmConfig::WasmConfig(const envoy::extensions::wasm::v3::PluginConfig& config)
       }
     }
   }
+
+  // Ensure that the Wasm extension's custom stat namespace is registered.
+  // We do that here because this is the only code path of
+  // all kinds of Wasm extension points.
+  ensureCustomStatNamespaceRegistered(Stats::getCustomStatNamespaces());
```

Comment:
Is it possible for us to change the signature of WasmConfig::WasmConfig to pass the `CustomStatNamepace&` in?